tensorflow/core/api_def/base_api/api_def_ROIAlign.pbtxt

```diff
@@ -0,0 +1,16 @@
+op {
+  graph_op_name: "ROIAlign"
+  summary: "      This op implements Region of Interest(RoI) align op as documented in arXiv:1703.06870."
+  description: <<END
```

Comment:
Use in_arg and out_srg to document inputs and outputs https://github.com/tensorflow/tensorflow/blob/273981699d92c1aa08ca52af99ee187ce5aaa5e1/tensorflow/core/framework/api_def.proto#L79

Reply:
This file is generated by bazel-bin/tensorflow/core/api_def/update_api_def . Is generating these files with update_api_def is wrong procedure? If so how can I generate this file correctly?

Reply:
Edit it manually after generating it.

I think the update_api_def script's purpose was to migrate the old REGISTER_OP stuff.

Reply:
Done